cmd/setup.go

```diff
@@ -59,6 +76,27 @@ func runSetupError(cmd *cobra.Command, args []string) error {
 		return microerror.Mask(err)
 	}
 
+	k8sApiUrl := os.Getenv(EnvVarK8sApiUrl)
+	k8sCert := os.Getenv(EnvVarK8sCert)
+	k8sCertCA := os.Getenv(EnvVarK8sCertCA)
+	k8sCertPrivate := os.Getenv(EnvVarK8sCertPrivate)
```

Comment:
does it make sense to have this imported via flags or rather via envs?
from e2e it doesnt really matter but for human use we would have flags which have really long values like
```
--cert-tls=Lsfaseoiqwfioonrbvowrbnwrobndosvvsdvsvoewoneovewovneoivnweovnewoivnewiovniweovnweiovnewoivnewiovnewiovnweoivnewiovnweoivvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvveiovniowrnvdknroiejabfdvuigehjkfvdehuivjdsnkreuvsdhjredcjrvhdinjsehufvdsnefhvsdnklevhndkjscfevijhdksxfhvdnkjsevhnijdsaclehvnsdjksdvnjkacsdvnjamrvasmrhvdawrhbuaschwedascnmhjvnsdkcsdvnkjclfsdvnkjmfvsdnkjcmsdvkjncmlsdvkjncmsdvnkjcmskdvnjmsdvjnkcmasdvnkjcmdvsnjkaclsdvjnk
 ```
as its double base64 encoded data
```

Reply:
IMO better flags, at the end we will see on the CI script something like `--cert-tls=${CERT_TLS}`.